kubernetes_state/check.py

```diff
@@ -181,6 +180,13 @@ def _label_to_tag(self, name, labels, tag_name=None):
         else:
             return None
 
+    def _remove_job_tag(self, name):
```

Comment:
It'd rename it to `_trim_job_tag`, as we are not removing it.
When rebasing, please keep it separate and called from the methods instead of hooking it in `_format_tag`